exporters/trace/src/test/java/com/google/cloud/opentelemetry/trace/TestSpanData.java

```diff
@@ -1,13 +1,5 @@
 /*
- * Copied from: https://github.com/open-telemetry/opentelemetry-java/blob/v0.8.0/testing_internal/src/main/java/io/opentelemetry/sdk/trace/TestSpanData.java
- *
- * This file should be updated in the future to reflect any changes to
- * the remote. This was only copied in the first place as the original
- * TestSpanData was moved here: https://github.com/open-telemetry/opentelemetry-java/pull/1512
- */
-
-/*
- * Copyright 2020, OpenTelemetry Authors
```

Comment:
This file should probably be excluded from the automatic Google copyright.

Reply:
No, it's required here too.